pynicotine/privatechat.py

```diff
@@ -213,16 +213,26 @@ def _message_user(self, msg, queued_message=False):
         """Server code 22."""
 
         username = msg.user
+        message = msg.msg
 
         if not queued_message:
             log.add_chat(_("Private message from user '%(user)s': %(message)s"), {
                 "user": username,
-                "message": msg.msg
+                "message": message
             })
 
             core.send_message_to_server(slskmessages.MessageAcked(msg.msgid))
 
-        if username != "server":
+        if username == "server":
+            start_str = "The room you are trying to enter ("
+
+            if message.startswith(start_str) and ") " in message:
+                # Show fake chat room message if join wasn't successful
```

Comment:
`# Show fake chat room message `... -> `# Refer message to chat room tab `...

There is nothing "fake" about this, it's a really sensible idea.

Reply:
It's fake in the sense that it's not an actual chat room message, but I suppose it can be interpreted as the original message itself being fake. I'll change the comment.

Reply:
> Not sure this specific change alone is worth mentioning

It is, because the user experience of this event is substantially altered.